src/MutationFunctions.jl

```diff
@@ -30,8 +30,8 @@ function mutateConstant(
         node = randomNode(tree)
     end
 
-    bottom = 0.1f0
-    maxChange = options.perturbationFactor * T + 1.0f0 + bottom
+    bottom = convert(T, 0.1)
```

Comment:
`0.1` does not have an example Float64 representation, so it might be worthwhile to have this as an exact constant `1//10` which converts to the best representation in the given type. Very minor detail though. IIRC the compiler should specialize on that during method specialization and essentially convert it before runtime.